src/__tests__/helpers.test.ts

```diff
@@ -0,0 +1,51 @@
+import { validateInput, shortenPublicHoliday } from "../helpers";
+import holidayMocks from "../__mocks__/mock.json";
+
+describe("Helpers tests", () => {
+  describe("validateInput helper tests", () => {
+    it.each([
+      { year: 2023, country: "GB", expected: true },
+      { year: 2023, country: "FR", expected: true },
+      { year: 2023, country: "DE", expected: true },
+      { year: 2023, country: "NL", expected: true },
+    ])(
+      "should validate user input year: $year and country: $country",
+      ({ year, country, expected }) => {
+        const result = validateInput({ year, country });
+        expect(result).toEqual(expected);
+      },
+    );
+
+    it.each([{ year: 2024 }, { year: 1 }, { year: 20233 }, { year: -102 }])(
+      "should throw error in case not valid year: $year passed",
+      ({ year }) => {
+        expect(() => validateInput({ year, country: "GB" })).toThrow(
```

Comment:
another way to test error handling
```suggestion
        await expect(validateInput({ year, country: "GB" })).rejects.toThrow(
```